sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -474,6 +474,35 @@ public async Task DownloadAsync_VersionId()
             // Assert
             Assert.IsNotNull(response.Value.Details.VersionId);
         }
+
+        [Test]
```

Comment:
Add TODO at the top to mark this as recorded only.

Reply:
is it possible to just `if` the body of the test until we have attribute for recorded only tests?